app/grandchallenge/core/views.py

```diff
@@ -102,40 +102,11 @@ def get_context_data(self, **kwargs):
                 url_title="Algorithms",
                 image="images/algorithms.png",
             ),
-            Highlight(
-                title="Find Certified Solutions",
-                bullet_points=[
-                    "Filter to easily find solutions to your clinical questions",
-                    "Compare product specifications",
-                    "Verify CE and FDA certification",
-                ],
-                url=reverse("products:product-list"),
-                url_title="Products",
-                image="images/products.png",
-            ),
```

Comment:
How does the feature overview on the hompage look now that we have 5 rather than 6 highlights?

Reply:
Unbalanced but we can add something back later

<img width="1160" alt="Screenshot 2024-02-26 at 15 36 09" src="https://github.com/comic/grand-challenge.org/assets/12661555/e734da40-5922-4c33-a40a-3de26e0c7945">

Reply:
I think it's best to leave it out as it is not longer a feature of GC. We used to have an alternating list which handled odd items better but that was removed in https://github.com/comic/grand-challenge.org/pull/2062.